### ISSUE TITLE
Establishing, or finding an alternative for "good old" schema.org/Role throughout Schema.org

### ISSUE THREAD

**Issue by l00mi** [schemaorg/schemaorg#2777]
For an internal project we need a reification of the schema:Organization to schema:Person relation mainly to specify the time validity of the relation.

There is https://schema.org/Role taking this place. But it seems not being used on the expected types where potentially applicable, which makes it difficult to find.

I like to ask what might be a way to work on this reification to make it more prominent?
* Adding to the "expected types" of the most probable properties like schema:employee, schema:actor, schema:athlete.
* Think about another reification all along.
* Not work on it as it seems more used for internal projects and there is no need for this reification for public purposes.
* ...

**Comment by dr-shorthair** [comment]
You might look at https://www.w3.org/TR/vocab-org/ for inspiration

**Comment by l00mi** [comment]
@dr-shorthair thank you for the pointer. I was actually weighting up FOAF and vocab-org, and schema.org, and have a slight tendency for schema.org. This because as of today it is more complete also on the more constraint use case of _FOAF_ and _vocab-org_. For an internal use-case the advantage of these over schema.org is the stronger definition of the relations.

Finally, I have at least a slight feeling that one of the authors of FOAF will invest more time in schema.org then in FOAF in the future (-:

**Comment by dr-shorthair** [comment]
Yeah - I wish they hadn't tied it to FOAF. It would almost have been better to have used `dcterms:Agent`.

**Comment by RichardWallis** [comment]
Check out the [conformance section](https://schema.org/docs/datamodel.html#conformance) of the Data Model documentation. It indicates that any property, in theory, can expect values Text, URL, Role in addition to those documented.

Having said that it is a feature, in respect of Role, that is not often utilised.

**Comment by l00mi** [comment]
Thank you. That is a very good hint. I was not aware of this section to be honest.

My question is would it be beneficial to add it also in chosen place as an _Expected Type_? A bit similar to the effort with the DefinedTerms.

It seems further while every where allowed by definition, `URL` is still on chosen places listed as _Expected Type_. I would expect that adding it to _Expected Type_ will make this construct more visible.

**Comment by RichardWallis** [comment]
Adding _Role_ to the expected type for some properties may turn into one of those never-ending tasks.

There may be some potential around [OrganizationRole](https://schema.org/OrganizationRole) and its subtype [EmployeeRole](https://schema.org/EmployeeRole) worth investigating.

**Comment by RichardWallis** [comment]
See issue #2787 for further discussion

**Comment by l00mi** [comment]
@RichardWallis I like to push this forward during summer. What would be a sensible way to approach this? Shall we start with:

* Adding a link in the description of http://schema.org/Role and https://schema.org/OrganizationRole https://schema.org/docs/datamodel.html#conformance 
* Adding as proposed by you in https://github.com/schemaorg/schemaorg/issues/2787  
  > Role subtypes - OrganizationRole, EmployeeRole, PerformanceRole, and possibly LinkRole - to the range of appropriate properties (alumniOf, worksFor, memberOf, actor, etc.). Also based upon your second example, there may be a case for a new Role subtype of OccupationRole.
* Integrating the essence (Role as intermediary / simple reification of any Property) from http://blog.schema.org/2014/06/introducing-role.html into either https://schema.org/docs/datamodel.html#conformance or the description of http://schema.org/Role.

I would be pleased if someone can guide me to this changes, shall I go for PRs directly, or does this need some more discussion?

**Comment by l00mi** [comment]
Can we have links in "descriptions"? Like on https://schema.org/hasOccupation

> The Person's occupation. jFor past professions, use Role for expressing dates.

**Comment by l00mi** [comment]
@RichardWallis shall I make a proposal to establish schema.org/Roles in the rest of the Schema Documentation?
